docker-compose.yaml

```diff
@@ -10,9 +10,9 @@ services:
       - 9001:9001
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
-      interval: 30s
+      interval: 5s
       timeout: 20s
-      retries: 3
+      retries: 18
```

Comment:
I would also be cool with updating this instead to 

```
      interval: 30s
      timeout: 20s
      retries: 3
      start_period: 10s
```

If you're not a fan of doing a 5s health check interval in perpetuity